### PR TITLE
feat: Image & Input

### PR DESCRIPTION
**Pull request by purohitdheeraj**
This Branch contains
- Image Component
- Input Component

### CODE REVIEW

File: library/components/images/image.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.responsive-image{
+    width: 100%;
+    height: auto;
+}
+
+.round-image{
+    border-radius: 100%;
+}
```
Comment (DivyanshSareen) [inline library/components/images/image.css]: can use aspect-ratio: 1/1 to avoid ellipse

File: library/components/input/input.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
+.form-field{
+    flex: 0 1 45rem;
+    background: var(--NEUTRAL);
+    padding: 2rem;
+    border-radius: 1rem;
+    box-shadow: 0px 0px 9px 3px rgba(41, 41, 41, 0.25);;
```
Comment (DivyanshSareen) [inline library/components/input/input.css]: use var for color

**Review by DivyanshSareen — APPROVED** [review]
Good code and documentation. Good to merge.

**Comment by Sahil25061999** [comment]
looks good to me can be merged

**Comment by purohitdheeraj** [comment]
> looks good to me can be merged

Hey @Sahil25061999 please approve it so that I can merge it

**Review by Sahil25061999 — APPROVED** [review]

**Pull request merged**